proto/osmosis/valset-pref/v1beta1/tx.proto

```diff
@@ -2,6 +2,7 @@ syntax = "proto3";
 package osmosis.valsetpref.v1beta1;
```

Comment:
Random idea that has nothing to do with this file:

Wouldn't it be cool to add to this module the ability for governance to create premade validator sets? For instance, governance can decide who should be in a validator set called "validators who relay" or "validators who review code".

Idk, maybe not a great idea, but was just considering governance sanctioned lists that frontends could ship with and make their own off these lists if they choose

Reply:
Resolving this as we talked about this in the call